pyretailscience/plots/line.py

```diff
@@ -76,16 +77,23 @@ def plot(
     """
     if x_col is not None and pd.api.types.is_datetime64_any_dtype(df[x_col]):
```

Comment:
Can you please combine this `if` and the `elif` together to reduce the cyclometric complexity.